src-vue/src/views/DeveloperView.vue

```diff
@@ -43,6 +47,24 @@ export default defineComponent({
                 position: 'bottom-right'
             });
         },
+        async checkLinuxCompatibility() {
+            let LinuxCompatible = await invoke("linux_checks");
+            if (!LinuxCompatible) {
+                ElNotification({
+                    title: 'Not linux compatible',
+                    message: 'GLIBC is not version 2.33 or greater',
+                    type: 'error',
+                    position: 'bottom-right'
+                });
+            } else {
+                ElNotification({
+                    title: 'Linux ompatible',
```

Comment:
Another one 😄

Reply:
![image](https://user-images.githubusercontent.com/99826409/194669291-593c3ac7-e905-45fc-b81b-7e9a9abe22b8.png)
ayo what
did i not push